crmd/main.c

```diff
@@ -138,6 +140,11 @@ crmd_init(void)
     state = s_crmd_fsa(C_STARTUP);
 
     if (state == S_PENDING || state == S_STARTING) {
+        if (safe_str_eq(start_state, "standby")) {
+            const crm_node_t * this_node = crm_get_peer(0, fsa_our_uname);
+            set_standby(fsa_cib_conn, this_node->uuid, NULL, "on");
+        }
+
```

Comment:
We should have a list of allowed values (probably "standby", "online" and "default"), and log a warning if the variable is set to something different. For example, if someone mistypes PCMK_node_start_state="stadnby", it might help them to see a log message like "Unrecognized start state 'stadnby', using default".